src/_langs/en/updates/2015-07-22-mediastream-deprecations.markdown

```diff
@@ -0,0 +1,155 @@
+</p>
+
+## Stop(), ended, active
+
+When the [W3C Working Group](http://w3c.github.io/mediacapture-main/getusermedia.html)
+looked at the problem of what happens when you add new tracks to a `MediaStream`,
+and whether an empty `MediaStream` is ended or not, they realized that there was
+no way to do `ended` (as in 'will never start again') on a `MediaStream` in
+a sensible way. In other parts of HTML5 'ended' means 'this has ended and will
+never resume'. 'Active' carries no such implication: an inactive stream can
+become active again, for instance if a new track is added to it. Rather than try
+to maintain a confusing attribute and function, the Working Group decided to
+remove it.
+
+Here's an example of how to use 'MediaStream.active' to check the status of a
+stream:
+
+    var gumStream;
+
+    navigator.getUserMedia({audio: false, video: true},
+        function(stream) {
+             gumStream = stream;
+            // ...
+        },
+        function(error) {
+            console.log('getUserMedia() error', error);
+        });
+
+    // …
+
+    if (gumStream.active) {
+        // do something with the stream
+    }
+
+Removing `stop()` from `MediaStream` did not remove any real functionality:
+processes for detaching source devices and so on have to be done on
+`MediaStreamTrack` anyway. Use `stop()` on `MediaStreamTrack` instead:
+
+    navigator.getUserMedia({audio: false, video: true},
+        function(stream) {
+             // can also use getAudioTracks() or getVideoTracks()
+            var track = stream.getTracks()[0];  // if only one media track
+            // ...
+            track.stop();
+        },
+        function(error){
+            console.log('getUserMedia() error', error);
+        });
+
+## label
+
+Turns out that nobody could quite figure out a use for this property!
+
+It had been added to the very first version of the spec, but nobody really knew
+what it was for, and it was completely unclear what happened to it when a stream
+was sent via `RTCPeerConnection`.
+
+The W3C Working Group [asked around](http://lists.w3.org/Archives/Public/public-html-media/2015Apr/0025.html),
+and nobody wanted it, so they removed it. Less things to support.
+
+To reiterate: `MediaStream.id` provides a unique identifier for a
+`MediaStream` and `MediaStreamTrack.label` provides the name of the source
+of a stream, such as the type of camera or microphone.
+
+- - -
+
+More information about `MediaStream` and `MediaStreamTrack` is available
+from [Mozilla Developer
+Network](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream), and
+HTML5 Rocks provides an excellent introduction to `getUserMedia()` in
+[Capturing Audio &
+Video](http://www.html5rocks.com/en/tutorials/getusermedia/intro/).
+
+As ever, we appreciate your feedback on changes to Chrome. You can follow
+[the](https://code.google.com/p/chromium/issues/detail?id=407039)
```

Comment:
Linking to a bug on the word 'the' will look like a mistake or people might assume it and the next word are linked to the same bug. Can you find another way to do this?